Docker/Exercise/06/compose.yaml

```diff
@@ -0,0 +1,24 @@
+services:
+  server:
+    build:
+      context: ./backend
+    command: npm run start
```

Comment:
Why is the command needed in this file too when its present in the Dockerfile too?